chipsec/module.py

```diff
@@ -72,29 +75,46 @@ def do_import(self):
                     self.logger.log_bad(traceback.format_exc())
                 raise msg
         return loaded
-
+    
+    def get_module_ids_dictionary(self):
+        with open(os.path.join(os.getcwd(), 'chipsec', 'library', 'module_ids.pkl'), 'rb') as module_ids_file:
+            module_ids = pickle.load(module_ids_file)
```

Comment:
Did you confirm pickle can be used in the shell?

Reply:
yes!